lib/benchee/formatters/console/run_time.ex

```diff
@@ -235,6 +235,27 @@ defmodule Benchee.Formatters.Console.RunTime do
   defp comparison_report([_scenario], _, _, _), do: []
   defp comparison_report(_, _, _, %{comparison: false}), do: []
 
+  defp comparison_report([scenario | other_scenarios], units, label_width, %{
+         reference_job: reference_job
+       })
+       when is_binary(reference_job) do
+    %{found_job: found_job, other_scenarios: other_scenarios} =
```

Comment:
Often in Elixir tuples are used in cases like this where a function returns more than one discrete "thing".

So, lines 242-250 could also be written something like this:

```elixir
{scenario, other_scenarios} =
  case find_reference_job(scenarios, reference_job) do
    {nil, other_scenarios} -> {scenario, other_scenarios}
    found -> found
  end
```